opm/simulators/timestepping/SimulatorTimer.cpp

```diff
@@ -42,14 +43,15 @@ namespace Opm
     {
         const int num_psteps = param.getDefault("num_psteps", 1);
         const double stepsize_days = param.getDefault("stepsize_days", 1.0);
+        const double end_step_ = std::numeric_limits<size_t>::max();
```

Comment:
This appears to be an unused local variable.  Is this statement supposed to assign a datamember instead?  The type `double` is also a little questionable given that the right-hand side is `size_t`.